### DISCUSSION TITLE
problem super cache with permalink utf8

### DISCUSSION THREAD

**Discussion by kh71** [pothi/wordpress-nginx#24] [Q&A]
thanks i try follow work all domain but have one domain use permalink postname.
please help

domain/ការទិញ​ខុន​ដូ​ពីសំណាក់/

check header show x-cache: MISS. if no phpfpm run site dead

**Comment by kh71** [comment]
yes i found problem

uri or request uri
string Uppercase but in server string  lowercase

how to convert var to lowercase

**Comment by pothi** [comment]
Hi @kh71 ,

Thanks for your time evaluating this repo for your use-case.

This is a generic issue with Nginx. Not particular to WP Super Cache plugin. I came across similar issues with utf8 on URLs. I use https://nginx.org/en/docs/njs/ to solve such issues. I hope that helps you too. Unfortunately, there is no other easy way to solve lowercase or utf8 problems within the configuration. If there is any, I am all ears.

Thanks again,
Pothi.

**Comment by kh71** [comment]
thanks reply but me not clear njs. can guide me.

i have searching in stackoverflow found but don't know how to addon perl too.

https://stackoverflow.com/questions/39795527/nginx-rules-for-upper-lower-case-locations
